moveit_experimental/moveit_jog_arm/src/collision_check_thread.cpp

```diff
@@ -52,7 +53,7 @@ CollisionCheckThread::CollisionCheckThread(const moveit_jog_arm::JogArmParameter
     ros::Duration(WHILE_LOOP_WAIT).sleep();
   }
 
-  planning_scene_monitor_.reset(new planning_scene_monitor::PlanningSceneMonitor(model_loader_ptr));
+  planning_scene_monitor_ = std::make_shared<planning_scene_monitor::PlanningSceneMonitor>(model_loader_ptr, std::make_shared<tf2_ros::Buffer>());
```

Comment:
I think this should be reverted.
Passing a TF buffer that is not connected to a TF listener, doesn't make sense.

The key is to avoid setting up an Octomap Monitor, because we don't have a valid TF listener.
I will prepare a corresponding fixup commit.